### ISSUE TITLE
head request fix

### ISSUE THREAD

**Issue by HongPong** [HongPong/broken-link-checker#43]
this patch may be worth doing via @pathduck

https://github.com/Pathduck/broken-link-checker/commit/2e754a01d2a9b3b78375ad7b5dac0d6e30b52182

**Comment by HongPong** [comment]
```
        if ( $nobody && !$result['timeout'] && !$use_get && ($result['broken'] || $result['redirect_count'] == 1)){
```

**Comment by Pathduck** [comment]
I've not really been sure if my logic here was sane - but I've not had any recurrences of false positives after I added this to my local installation 😄 👍
